.github/workflows/pr_comment_bot.yml

```diff
@@ -199,7 +199,7 @@ jobs:
 
       # Check if the PR build/test needs to run
       - name: Checkout
-        if: ${{ steps.check_command.outputs.result == 'run-tests' || steps.check_command.outputs.result == 'test-force-approve' }}
+        if: ${{ steps.check_command.outputs.result == 'run-tests' || steps.check_command.outputs.result == 'test-force-approve' || steps.check_command.outputs.result == 'test-destroy-env' }}
```

Comment:
That's over 80 characters on a line . . . :-)

Reply:
Yeah, I've got a number of tidy-up tasks noted down (currently testing splitting this somewhere else) 😄